### PR TITLE
feat: added aria-live as a alias for accessibility-live-region

### PR DESCRIPTION
**Pull request by mayank-96**
## Summary
This adds `aria-live` alias for `accessibilityLiveRegion`, it unifies aria-live and accessibilityLiveRegion and also maps `aria-live='off'` to `accessibilityLiveRegion='none'` as requested on https://github.com/facebook/react-native/issues/34424

## Changelog

[General][Added] - Added aria-live alias for accessibilityLiveRegion.

## Test Plan

```js
<View aria-live="polite">
  <Text>Clicked {this.state.count} times</Text>
</View>

<View aria-live="off">
  <Text>Clicked {this.state.count} times</Text>
</View>
```

### CODE REVIEW

File: Libraries/Components/Pressable/Pressable.js [hunk 1]
```diff
@@ -52,6 +52,7 @@ type Props = $ReadOnly<{|
   accessibilityValue?: ?AccessibilityValue,
   accessibilityViewIsModal?: ?boolean,
   accessible?: ?boolean,
+  'aria-live'?: ?('none' | 'polite' | 'assertive' | 'off'),
```
Comment (necolas) [inline Libraries/Components/Pressable/Pressable.js:55]: `'none'` is not a valid value for `aria-live`

File: Libraries/Components/Touchable/TouchableWithoutFeedback.js [hunk 2]
```diff
@@ -42,6 +42,7 @@ type Props = $ReadOnly<{|
   accessibilityValue?: ?AccessibilityValue,
   accessibilityViewIsModal?: ?boolean,
   accessible?: ?boolean,
+  'aria-live'?: ?('none' | 'polite' | 'assertive' | 'off'),
```
Comment (necolas) [inline Libraries/Components/Touchable/TouchableWithoutFeedback.js:45]: Remove 'none'

File: Libraries/Components/View/ViewPropTypes.js [hunk 3]
```diff
@@ -292,6 +292,16 @@ type AndroidViewProps = $ReadOnly<{|
...
 
+  /**
+   * Indicates to accessibility services whether the user should be notified
+   * when this view changes. Works for Android API >= 19 only.
+   *
+   * @platform android
+   *
+   * See https://reactnative.dev/docs/view#accessibilityliveregion
+   */
+  'aria-live'?: ?('none' | 'polite' | 'assertive' | 'off'),
```
Comment (necolas) [inline Libraries/Components/View/ViewPropTypes.js:303]: Remove 'none'

**Review by necolas — CHANGES_REQUESTED** [review]

File: Libraries/Components/Touchable/TouchableWithoutFeedback.js (hunk 2 above)
Comment (mayank-96) [reply Libraries/Components/Touchable/TouchableWithoutFeedback.js]: Updated the requested changes.

File: Libraries/Components/Pressable/Pressable.js (hunk 1 above)
Comment (mayank-96) [reply Libraries/Components/Pressable/Pressable.js]: Updated the requested changes.

File: Libraries/Components/View/ViewPropTypes.js (hunk 3 above)
Comment (mayank-96) [reply Libraries/Components/View/ViewPropTypes.js]: Updated the requested changes.

**Review by necolas — APPROVED** [review]

**Comment by mayank-96** [comment]
Hello @necolas, Let me know if anything else needs to be done here.

**Pull request closed without merging**